libs/aws/langchain_aws/utils.py

```diff
@@ -48,3 +51,83 @@ def get_token_ids_anthropic(text: str) -> List[int]:
     tokenizer = client.get_tokenizer()
     encoded_text = tokenizer.encode(text)
     return encoded_text.ids
+
+
+def get_aws_client(
+        region_name: str = None,
+        credentials_profile_name: str = None,
+        aws_access_key_id: SecretStr = None,
+        aws_secret_access_key: SecretStr = None,
+        aws_session_token: SecretStr = None,
+        endpoint_url: str = None,
+        config: Any = None,
+        service_name: str = None,
```

Comment:
service_name shouldn't be optional, I would also move it to the top as first parameter.
```suggestion
        service_name: str,
```

Reply:
Done 👍